config.default

```diff
@@ -76,6 +76,9 @@ autocloseFileLimitFraction = 0.5
 # with a single time slice.
 maxChunkSize = 10000
 
+# Reasonable size for an array to be cached into memory
+reasonableArraySizeGB = 3.0
+
```

Comment:
Nice addition.  I assume your plan was to use this in places other than MOC down the line?